db/schema.rb

```diff
@@ -163,6 +163,17 @@
     t.index ["user_id"], name: "index_invite_statuses_on_user_id"
   end
 
+  create_table "lti_configurations", force: :cascade do |t|
```

Comment:
Not sure why everytime I pull master, this file always comes back. But I'll fix this 😄